local-stack/console-values.yaml

```diff
@@ -3,9 +3,10 @@
 # Console configuration
 ######
 config:
-  platform:
-    https:
-      existingSecret: "custom-console-crt-secret" # Secret with TLS certificate tls.crt and tls.key PEMs
+  # If you need TLS passthrough to the Console server
+  # platform:
+  #   https:
+  #     existingSecret: "custom-console-crt-secret" # Secret with TLS certificate tls.crt and tls.key PEMs
```

Comment:
this go against the recommendation of using TLS down to the containers. 
Not tested but we should be able to have a TLS on ingress using cert-manager annotation and still keep HTTPS to expose console API with an internal TLS certificate. 
It need to add ingress annotations : 
```
    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
 ```

Reply:
This works! thanks! will push soon